sdk/src/client/node_api/core/routes.rs

```diff
@@ -93,19 +93,16 @@ impl ClientInner {
 
     // Blocks routes.
 
-    /// Returns tips that are ideal for attaching a block.
-    /// GET /api/core/v3/tips
-    pub async fn get_tips(&self) -> Result<Vec<BlockId>> {
-        let path = "api/core/v3/tips";
+    /// Returns information that are ideal for attaching a block.
+    /// GET /api/core/v3/blocks/issuance
+    pub async fn get_issuance(&self) -> Result<IssuanceBlockHeaderResponse> {
+        const PATH: &str = "api/core/v3/blocks/issuance";
```

Comment:
In case you are interested, there are more places where you can made it const PATH

Reply:
Sure